bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java

```diff
@@ -195,30 +196,30 @@ public boolean equals(Object obj) {
   private SerializableScan serializableScan;
 
   /**
-   * Creates a {@link CloudBigtableScanConfiguration} using the specified information.
+   * Creates a {@link CloudBigtableScanConfiguration} using the specified configuration.
```

Comment:
I think it is reasonable to describe together all the ID paramenters into "location" or "table", and separetely the scan, which acts on the location specified by the others.

Up to you.